apps-rendering/src/components/Follow/index.tsx

```diff
@@ -56,19 +95,40 @@ const Follow: FC<Props> = ({ contributors, format }) => {
 		format.theme !== ArticleSpecial.Labs
 	) {
 		return (
-			<button
-				className="js-follow"
-				css={styles(format)}
-				data-id={contributor.id}
-				data-display-name={contributor.name}
-			>
-				<span className="js-follow-status" css={followStatusStyles}>
-					<FollowStatus
-						isFollowing={false}
-						contributorName={contributor.name}
-					/>
-				</span>
-			</button>
+			<>
+				<button
+					className="js-follow-tag"
+					css={styles(format)}
+					data-id={contributor.id}
+					data-display-name={contributor.name}
+				>
```

Comment:
I think we should not leave a button in the DOM that might not mean anything (e.g if `MyGuardian` is not enabled) as it could still get picked up by the user's accessibility features like a screen reader.

Is there a way we could remove the button until we know it's required?